bots/pr/src/main/java/org/openjdk/skara/bots/pr/TagCommand.java

```diff
@@ -63,7 +63,10 @@ public void handle(PullRequestBot bot, HostedCommit commit, LimitedCensusInstanc
             Path scratchPath, CommandInvocation command, List<Comment> allComments, PrintWriter reply) {
         try {
             if (censusInstance.contributor(command.user()).isEmpty()) {
-                reply.println("Only OpenJDK [contributors](https://openjdk.org/bylaws#contributor) can use the `/tag` command.");
+                reply.println("To use the `/tag` command, you need to be in the OpenJDK [census](https://openjdk.org/census)"
+                        + " and your GitHub account needs to be linked with your OpenJDK username"
+                        + "([how to associate your GitHub account with your OpenJDK username]"
+                        + "(https://wiki.openjdk.org/display/skara#Skara-AssociatingyourGitHubaccountandyourOpenJDKusername)).");
```

Comment:
Since we are only allowing integrators to use the tag command, then this message may be unnecessary. It could still serve as a safety net that the integrators are actually also in the census. For that we should move this check to after the integrators check. Also the same space is missing here after "username".

Reply:
> Since we are only allowing integrators to use the tag command, then this message may be unnecessary. It could still serve as a safety net that the integrators are actually also in the census. For that we should move this check to after the integrators check. Also the same space is missing here after "username".

Yes, it makes sense, thx for the suggestion!